src/NzbDrone.Core/Notifications/Discord/DiscordFieldType.cs

```diff
@@ -11,7 +11,8 @@ public enum DiscordGrabFieldType
         Links,
         Release,
         Poster,
-        Fanart
+        Fanart,
+        Indexer
```

Comment:
Not sure if there is a specific order these should be in

Reply:
Nope, at the end keeps them in the same order.